lgc/test/lgcdis-table_jump.lgc

```diff
@@ -0,0 +1,50 @@
+; RUN: llvm-mc -triple=amdgcn--amdpal -mcpu=gfx1030 -filetype=obj %s | \
```

Comment:
I don't see the disassembler file in the coverage report. Are we sure that this test is actually executed?

Reply:
The test count seems correct, but this warning looks a bit suspicious. Any chance the coverage data is not being properly updated?

https://github.com/GPUOpen-Drivers/llpc/runs/5756068094?check_suite_focus=true
```
warning: 79 functions have mismatched data
```

Reply:
The coverage report does not currently cover LLVM due to size limitations. Since the disassembler is used as a LLVM library, it will not appear in the report.

Reply:
@vettoreldaniele :^

Reply:
@vettoreldaniele would it be possible to include lgc but not the rest of llvm? How difficult would that be?

Reply:
After checking, yes it is possible. This is outside of LLVM so it will work. The reason it's not included at the moment is that we simply don't have `lgcdis` in the binaries analyzed by the coverage report generator. We only have `amdllpc` currently. I will submit a patch to add this. We should probably include other binaries that are run in these tests to make sure they're accounted for. Anything outside LLVM is easy to handle.

Reply:
Looks like it's #1756.